src/getting-started/guides.md

```diff
@@ -1,7 +1,6 @@
 # Guides
-The subsequent guides are separated into two distinct categories depending on the user's usage intention.
-
-The following sections are available:
+Our user guides are separated into different categories depending on the user's usage intention and. The following
```

Comment:
I would like to avoid phrases like "we" & "our" and rather use the third person or general phrases.